app/src/main/java/com/ataulm/whatsnext/Navigator.java

```diff
@@ -20,9 +20,9 @@ public void navigateToSearch() {
         activity.startActivity(intent);
     }
 
-    public void navigateToFilm(String letterBoxd) {
+    public void navigateToFilm(FilmSummary filmSummary) {
         Intent intent = new Intent(activity, FilmActivity.class);
-        intent.putExtra(FilmActivity.EXTRA_FILM_ID, letterBoxd);
+        intent.putExtra(FilmActivity.EXTRA_FILM_SUMMARY, filmSummary);
```

Comment:
This lets us show the content we do have immediately while we load the full film.

Reply:
We could show it in a more fun way though - like incorporate the data we do have into a loading screen. This could be a good opportunity to try MotionLayout.